code/sorting/src/bogo_sort/bogo_sort.c

```diff
@@ -0,0 +1,42 @@
+/* Part of cosmos by OpenGenus Foundation  */
+
+/*
+* bogo_sort.c
+* created by Riya
+*/
+
+#include <stdio.h>    
+#include <stdlib.h>    
+
+int main()    

```

Comment:
Function return type and definition should be in separate lines
`int'
'main()'